tests/furusiya/model/skills/test_whirlwind.py

```diff
@@ -0,0 +1,58 @@
+from unittest.mock import Mock
+
+import math
+import pytest
+
+from model.skills.whirlwind import Whirlwind
+from model.systems.fighter_system import FighterSystem
+
+
+def _mock_factory(*args, **kwargs):
+    m = Mock(*args, **kwargs)
+    m.distance.side_effect = lambda x, y: math.sqrt((x - m.x) ** 2 + (y - m.y) ** 2)
+    return m
+
+
+@pytest.fixture
+def player():
+    yield _mock_factory(x=1, y=2)
+
+
+@pytest.fixture
+def player_fighter():
+    yield Mock()
+
+
+@pytest.fixture
+def bushslime():
+    yield _mock_factory(x=7, y=8)
+
+
+@pytest.fixture
+def _fighter():
+    yield Mock()
+
+
+@pytest.fixture
+def tigerslash():
+    yield _mock_factory(x=2, y=2)
```

Comment:
Can you use relative coordinates here? That would make it clearer.

Reply:
Relative coordinates? Relative to what, exactly?